package.json

```diff
@@ -52,6 +52,8 @@
   ],
   "devDependencies": {
     "@solid/community-server": "^5.1.0",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^14.4.3",
```

Comment:
A recommended utility library on top of React Testing Library. Provides a lot of functions that simulate how a user actually interacts with a system. E.G. `user.type(input,'hello')` simulates the user mousing over the `input` element, clicking on it, then typing h-e-l-l-o on their keyboard.